move-mutation-test/Cargo.toml

```diff
@@ -4,6 +4,7 @@ version = "0.1.0"
 description = "Move mutation unit testing tool"
 authors.workspace = true
 edition.workspace = true
+homepage.workspace = true
```

Comment:
I added these since I saw that `cargo new` created this one for `mutator-common` crate